blocks/accordion/accordion.css

```diff
@@ -22,6 +22,7 @@
   font-size: 18px;
   line-height: 27px;
   color: #000;
+  margin-bottom:15px
```

Comment:
not sure if this is need to be set on the `.accordion-item-header`. On the bitdefender website the accordion header does not have a margin
![image](https://github.com/hlxsites/bitdefender/assets/26941572/9845aefd-3f5c-47a9-ad9a-b68564247c0c)
because of the margin on the header 
![image](https://github.com/hlxsites/bitdefender/assets/26941572/ae1ca448-d378-4e17-b0bc-fe94ba80d95c)
the accordion item is bigger than on the bitdefender website
![image](https://github.com/hlxsites/bitdefender/assets/26941572/a9d998e3-9c9d-4e01-b199-ba53d914992e)

Reply:
our implementation is not using headers, but only text

Reply:
yes, but without the margin, the text and the arrow are in the middle of the accordion item

Reply:
Reverted the p margin changes for now - so p has a top and bottom margin as before.